docker/dev-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+# prevent bundle errors on missing gems
+bundle check || bundle install --jobs 20 --retry 5
```

Comment:
I wonder if `./bin/setup` would be a useful addition here? It does the bundle check business too 🤷‍♂️

Reply:
Ah, must have been a Rails 5.x thing. I think we should remove it in favor of setup?